andela/travel_tool_front
# #162375484 Tests for adding a checklist item (e2e)

#### What does this PR do?
End-to-End testing for adding a travel checklist item.

#### Description of Task to be completed?
* Add test for input validation.
* Add test for a checklist item validation
* Check that if a checklist item that exists is added again an error message will appear

#### How should this be manually tested?
* Clone the repo `git clone https://github.com/andela/travel_tool_front.git`
* Change directory with `cd travel_tool_front`
* Checkout to the branch `ch-e2e-travelchecklist-162375484` on frontend and `develop` on the backend
* Run `yarn start:dev` to start the backend server.
* Run the cypress test with `yarn end2end`
* A new window will open with the test files
* Click on `create-travelchecklist.spec.js` under the Travelchecklist folder.
* The e2e tests should run successfully. 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162375484](https://www.pivotaltracker.com/story/show/162375484)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
